tiny_dnn/config.h

```diff
@@ -31,6 +31,11 @@
 // #define CNN_USE_OMP
 
 /**
+ * define to enable Grand Central Dispatch parallelization
+ */
+//#define CNN_USE_GCD
```

Comment:
@rolandpersson See comment by @edgarriba , you need to add couple of lines to CMake